src/frontend/typer/sanity/modaler/andreUtbetalinger.ts

```diff
@@ -1,6 +1,7 @@
-import { LocaleRecordString } from '../sanity';
+import { LocaleRecordBlock, LocaleRecordString } from '../sanity';
 
 export interface IAndreUtbetalingerTekstinnhold {
     leggTilPeriodeForklaring: LocaleRecordString;
     flerePerioder: LocaleRecordString;
+    leggTilFeilmelding: LocaleRecordBlock;
```

Comment:
Hvorfor hete disse `leggTilFeilmelding` og ikke bare `feilmelding`? Skal brukeren selv legge inn feilmeldinger?

Reply:
De tekstene som starter med `leggTil` hører til funksjonalitet for å legge til f.eks. en periode. Brukeren skal altså ikke legge til en knapp eller forklaring. Istedenfor hører `leggTilKnapp` og `leggTilForklaring` til funksjonalitet for å legge til noe.

Det samme gjelder feilmelding. Det er en feilmelding knyttet til det å legge til noe.

Strukturen på dette er ikke den mest selvforklarende, men ble gjort slikt for å følge `leggTilKnapp` tekstene som har ligget der lenge.